draft-wang-ppm-differential-privacy.md

```diff
@@ -331,10 +350,6 @@ This document uses the same conventions for error handling as {{!DAP}}.
 Let `exp(EPSILON)` denote raising the numeric constant `e` to the power of
 `EPSILON`.
 
-Under Central Limit Theorem (CLT), the sum of a large number of independently
```

Comment:
From Kunal: "in general, the rate at which CLT gives convergence to Gaussian is not good enough to get reasonable (eps, delta)-DP guarantees. While it may provide intuition, if we want to use, say, sum of binomials to get something like a Gaussian, we would want to analyze it from scratch."